### ISSUE TITLE
cloudflare_firewall_rule bypass

### ISSUE THREAD

**Issue by Subetov** [terraform-providers/terraform-provider-cloudflare#567]
### Terraform Version
Terraform v0.12.12

### Affected Resource(s)
cloudflare_firewall_rule

### Terraform Configuration Files
```hcl
resource "cloudflare_firewall_rule" "firewall_rule_c9d2553ef0b543b69a1325ccf5a729f0" {
  provider = cloudflare

  zone_id = cloudflare_zone.cloudflare.id

  description = "pingdom_user_agent_allow"
  filter_id = cloudflare_filter.filter_pingdom_useragent.id
  action = "bypass"
}
```

### Debug Output
```hcl
Error: expected action to be one of [block challenge allow js_challenge log], got bypass
  on cloudflare.tf line 1166, in resource "cloudflare_firewall_rule" 

"firewall_rule_c9d2553ef0b543b69a1325ccf5a729f0":
1166: resource "cloudflare_firewall_rule" "firewall_rule_c9d2553ef0b543b69a1325ccf5a729f0" {
```
### Expected Behavior
https://developers.cloudflare.com/firewall/cf-firewall-rules/actions/

Create firewall_rule with action "Bypass"

### Actual Behavior
Resource creation fails

**Comment by jacobbednarz** [comment]
Happy to review a PR for this feature.

**Comment by garrettgalow** [comment]
#568 adds this

**Comment by patryk** [comment]
Fixed by #568
